src/api/core/warren/conf.py

```diff
@@ -46,6 +46,7 @@ class Settings(BaseSettings):
     ALLOWED_HOSTS: List[str] = [
         "http://localhost:8090",
     ]
+    ALLOW_ORIGIN_REGEX = r"https://.*\.ngrok-free\.app"
```

Comment:
What about allowing this origin from the `WARREN_ALLOWED_HOSTS` environment variable (as it's only useful for development purpose)?

```suggestion
    ALLOWED_HOSTS: List[str] = [
        "http://localhost:8090",
    ]
```

Reply:
I wanted to, but it seems unlikely to work. I checked the [doc](https://fastapi.tiangolo.com/tutorial/cors/)—allow_origins doesn't support regex. I tried again, but it still triggers a CORS issue. 😕

Reply:
If you pass a static list via the environment (not a regex) it still does not work?

Reply:
Done, WDYT?